challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_pick_item.py

```diff
@@ -112,4 +112,4 @@ def __init__(self, robot, pick_spot_id, pick_spot_navigation_area):
     rospy.init_node(os.path.splitext("test_" + os.path.basename(__file__))[0])
     robot_instance = get_robot("hero")
     robot_instance.reset()
-    NavigateToAndPickItem(robot_instance, "dinner_table", "in_front_of").execute()
+    NavigateToAndPickItem(robot_instance, "kitchen_table", "in_front_of").execute()
```

Comment:
This fix was due to the entity id in bordeaux. We should keep this dinner table for in the lab